crates/frontend/src/components/table.rs

```diff
@@ -7,6 +7,57 @@ use leptos::*;
 use serde_json::{json, Map, Value};
 use superposition_types::SortBy;
 
+#[component]
+pub fn expandable_text(
+    value: String,
+    max_length: usize,
+    formatter: types::CellFormatter,
+    row: Map<String, Value>,
+    class_name: String,
+    is_expandable: bool,
+) -> impl IntoView {
+    let value = StoredValue::new(value);
+    let get_child = move |val: &str| (formatter)(val, &row);
+    let (is_expanded_rs, set_is_expanded_ws) = create_signal(false);
```

Comment:
since the setter already has `set` in the name, we can drop the `_rs` and `_ws` part, right ?
```suggestion
    let (is_expanded, set_is_expanded) = create_signal(false);
```